data/_ui/tooltips.txt

```diff
@@ -667,10 +667,10 @@ tip "shield protection:"
 	`Protection against shield damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in shield damage, while a total value of 11 only results in a 10 percent reduction.`
 
 tip "silent jumps:"
-	`Prevents a ship from making hyperdrive or jump drive sounds when entering or leaving a system.`
+	`Prevents a ship from emitting sounds when entering or leaving a system via hyperjump.`
 
 tip "silent scans:"
-	`Prevents a ship from generating scan sounds.`
+	`Prevents a ship from emitting sounds while scanning other ships.`
```

Comment:
```suggestion
	`Prevents a ship from emitting scan sounds.`
```
Now here, emitting would be fine, though why make the tip longer? It's sufficient to just change the one word - this stops scan sounds in general, compared to the above, there's no reason to specify other ships when scanning is the only case this occurs.

Reply:
Hmm, well for these tooltips I was applying suggestions from bene, who was hoping to have them from more of an in-universe perspective.

Reply:
The way the suggestion is worded, "prevents a ship from emitting sounds while scanning other ships", isn't specific to scanning. It can be interpreted as preventing a ship from emitting ANY sound while scanning other ships; whereas the original is specific in meaning scanner sounds.

Reply:
Actually that is a fair call.